src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj

```diff
@@ -15,16 +15,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.4" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
     <PackageReference Include="System.Xml.XPath" Version="4.0.1" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.3" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
     <PackageReference Include="System.Text.RegularExpressions" Version="4.3.0" />
     <PackageReference Include="System.Xml.XPath.XDocument" Version="4.0.1" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
```

Comment:
This shouldn't be needed as it is already included on line 21

Reply:
same for this

Reply:
See my other comment. The `ItemGroup` at line 17 applies to all TFMs. In fact, this project only supports `netstandard2.0` - there isn't even a netstandard1.4 or net46 TFM anymore

Reply:
Ya this makes more sense and will cleanup this file. Thanks for the suggestion.